src/convertToObject.js

```diff
@@ -1,12 +1,25 @@
 'use strict';
-
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const newArr = sourceString
```

Comment:
There is an issue with the assignment of `newArr`. The code `sourceString` should be followed by a semicolon to properly terminate the statement. This will prevent potential syntax errors.